.changes/v2.25.0/680-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Patched `VdcGroup.CreateDistributedFirewallRule` method that return incorrect single rule when
```

Comment:
```suggestion
* Patched `VdcGroup.CreateDistributedFirewallRule` method that returned incorrect single rule when
```
Since we have fixed it, it doesn't return the incorrect data anymore.

Reply:
Fixed. Thanks